python/lsst/ts/externalscripts/base_make_calibrations.py

```diff
@@ -26,6 +26,8 @@
 import asyncio
 import collections
 import os
+import time
+import numpy as np
```

Comment:
Please, add an empty line between these import statements

Reply:
Is there a guideline somewhere as to why it should be like this? (I added the line between "time" and "numpy" imports, I'm just curious).

Reply:
that's a good question.. I'm not sure..